kernel/packages/shared/catalogs/sagas.ts

```diff
@@ -162,8 +162,15 @@ function* fetchWearablesV2(filters: WearablesRequestFilters) {
   if (filters.ownedByUser) {
     if (WITH_FIXED_COLLECTIONS) {
       const collectionIds = WITH_FIXED_COLLECTIONS.split(',')
```

Comment:
Can we add a comment here as a disclaimer showing that this must be used only in zone for testing purposes?

Reply:
Done!